packages/mobile/src/components/comments/CommentBlock.tsx

```diff
@@ -144,26 +162,28 @@ export const CommentBlockInternal = (
                     ) : null}
                   </Flex>
                 </Flex>
-              ) : null}
-              {userId !== undefined ? (
-                <CommentBadge
-                  isArtist={isCommentByArtist}
-                  commentUserId={userId}
-                />
-              ) : null}
-            </Flex>
-            <CommentText
-              isEdited={isEdited && !isTombstone}
-              isPreview={isPreview}
-              commentId={commentId}
-              mentions={mentions}
-              trackDuration={track.duration}
-              navigation={navigation}
-              onCloseDrawer={closeDrawer}
-            >
-              {message}
-            </CommentText>
+                {!isPreview &&
```

Comment:
nit would it make sense to pull these out into a var? `shouldShowCommentBadge = !isPreview && ...`

Reply:
great suggestions all around!